src/controllers/handlers/name-owner-handler.ts

```diff
@@ -0,0 +1,14 @@
+import { HandlerContextWithPath, NameOwner } from '../../types'
+
+export async function nameOwnerHandler(
+  context: HandlerContextWithPath<'nameOwnerFetcher' | 'logs', '/users/names/:name/owner'>
+): Promise<{ status: 200; body: NameOwner }> {
+  const { name } = context.params
+  const { nameOwnerFetcher } = context.components
+
```

Comment:
Would you mind validating the received name? If we're accepting names as `myName.dcl.eth` we need to convert them into `myName` in order to use in the graph query.